docs/pages/2020_BIGMAP/sections/basics.rst

```diff
@@ -3,3 +3,658 @@
+Note that by executing the ``multiply`` function, all input and output nodes are automatically stored in the database:
+
+.. code-block:: ipython
+
+    In [5]: y
+    Out[5]: <Int: uuid: 7865c8ff-f243-4443-9233-dd303a9be3c5 (pk: 2) value: 3>
+
+We had not yet stored the data node assigned to the ``y`` variable, but by providing it as an input argument to the ``multiply`` function, it was automatically stored with PK = 2.
+Similarly, the returned ``Int`` node with value 6 has been stored with PK = 4.
+
+Let's once again leave the IPython shell with ``exit()`` and look for the process we have just run using the ``verdi`` CLI:
+
+.. code:: console
+
+    $ verdi process list
+
+The returned list will be empty, but don't worry!
+By default, ``verdi process list`` only returns the *active* processes.
+If you want to see *all* processes (i.e. also the processes that are *terminated*), simply add the ``-a`` option:
+
+.. code:: console
+
+    $ verdi process list -a
+      PK  Created    Process label    Process State    Process status
+    ----  ---------  ---------------  ---------------  ----------------
+       3  1m ago     multiply         ⏹ Finished [0]
+
+    Total results: 1
+
+    Info: last time an entry changed state: 1m ago (at 09:01:05 on 2020-05-13)
+
+We can see that our ``multiply`` calculation function was created 1 minute ago, assigned the PK 3, and has ``Finished``.
+
+As a final step, let's have a look at the provenance of this simple calculation.
+The provenance graph can be automatically generated using the verdi CLI.
+Let's generate the provenance graph for the ``multiply`` calculation function we have just run with PK = 3:
+
+.. _BIGMAP_2020_Basics:calcfunction:graph:
+
+.. code-block:: console
+
+  $ verdi node graph generate 3
+
+The command will write the provenance graph to a ``.pdf`` file.
+Use your favorite PDF viewer to have a look.
+It should look something like the graph shown in :numref:`BIGMAP_2020_Basics_fig_calcfun_graph`.
+
+.. _BIGMAP_2020_Basics_fig_calcfun_graph:
+.. figure:: include/images/calcfun_graph.png
+    :scale: 50
+    :align: center
+
+    Provenance graph of the ``multiply`` calculation function.
+
+.. note:: Remember that the PK of the ``calcfunction`` can be different for your database.
+
+.. _tutorial:basic:calcjob:
+
+CalcJobs
+========
+
+When running calculations that require an external code or run on a remote machine, a simple calculation function is no longer sufficient.
+For this purpose, AiiDA provides the ``CalcJob`` process class.
+
+To run a ``CalcJob``, you need to set up two things: a ``code`` that is going to implement the desired calculation and a ``computer`` for the calculation to run on.
+
+If you're running this tutorial in the Quantum Mobile VM or on Binder, these have been pre-configured for you. If you're running on your own machine, you can follow the instructions in the panel below.
```

Comment:
Just a reminder that the code is not pre-configured in the VM's, so this part needs to be adapted.